front/app/containers/Admin/reporting/components/ReportBuilder/Widgets/SingleIdeaWidget/IdeaCard/useTextNumberOfVotes/index.ts

```diff
@@ -0,0 +1,32 @@
+import { IPhaseData } from 'api/phases/types';
+import useLocalize from 'hooks/useLocalize';
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+const useTextNumberOfVotes = ({
```

Comment:
Shouldn't we create something like this much higher in the component hierarchy and reuse it in all (4 or more) places where we need it?